packages/next-swc/crates/next-core/src/next_shared/resolve.rs

```diff
@@ -0,0 +1,86 @@
+use std::collections::HashSet;
+
+use anyhow::Result;
+use lazy_static::lazy_static;
+use turbo_binding::{
+    turbo::tasks_fs::FileSystemPathVc,
+    turbopack::core::{
+        issue::unsupported_module::UnsupportedModuleIssue,
+        resolve::{
+            parse::{Request, RequestVc},
+            pattern::Pattern,
+            plugin::{ResolvePlugin, ResolvePluginConditionVc, ResolvePluginVc},
+            ResolveResultOptionVc,
+        },
+    },
+};
+use turbo_tasks_fs::glob::GlobVc;
+
+lazy_static! {
+    static ref UNSUPPORTED_PACKAGES: HashSet<String> = ["@vercel/og".to_owned()].into();
+    static ref UNSUPPORTED_PACKAGE_PATHS: HashSet<(String, String)> =
+        [("@next/font".to_owned(), "/local".to_owned())].into();
```

Comment:
We added `@next/font/local` support in https://github.com/vercel/next.js/pull/47369.

Reply:
Cool, updated https://github.com/vercel/next.js/pull/48837/commits/e1d0367fc7f493314d8c3576da45b09c4496c4eb